common/components/Footer/index.tsx

```diff
@@ -92,7 +93,21 @@ const LINKS_SOCIAL = [
 ];
 
 export default class Footer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { isOpen: false };
+  }
+
+  public toggleModal = () => {
+    this.setState({ isOpen: !this.state.isOpen });
+    document.body.classList.toggle('no-scroll');
+  };
+
   public render() {
+    const buttons: IButton[] = [
```

Comment:
Is it necessary to type these? Or will typescript just know that the shape is 👌

Reply:
Nope not necessary, still getting familiar with TypeScript, I'll update this.

Reply:
Yes it's necessary. TD needs the shape of buttons defined for the modal.